src/ray/object_manager/object_buffer_pool.h

```diff
@@ -166,13 +177,19 @@ class ObjectBufferPool {
   /// Returned when GetChunk or CreateChunk fails.
   const ChunkInfo errored_chunk_ = {0, nullptr, 0, nullptr};
 
-  /// Mutex on public methods for thread-safe operations on
-  /// get_buffer_state_, create_buffer_state_, and store_client_.
-  mutable std::mutex pool_mutex_;
   /// Determines the maximum chunk size to be transferred by a single thread.
   const uint64_t default_chunk_size_;
+
+  /// Mutex to protect create_buffer_ops_ and create_buffer_state_.
+  mutable std::mutex pool_mutex_;
```

Comment:
Can you change this to use absl condition var support instead of std::condition_variable? https://abseil.io/docs/cpp/guides/synchronization